arch/c4_context.md

```diff
@@ -0,0 +1,49 @@
+# Level 1: Radix System Context
+
+This page implement the first "C" in the [C4Model](https://c4model.com/#coreDiagrams): the **C**ontext Model.
+
+All production Systems (External and Radix) and Users that are commonly found on Radix networks are introduced on this page.
+
+![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
+
+# Shards and Chunks
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/), shards enable Radix Networks to scale infinitely.
+This is achieved by splitting up global state (Ledger) to different nodes. Each node can be part of one or more Shards slices - called Chunks.
```

Comment:
splitting up global state *in a deterministic way* - the key part is that we need to be able to tell *where* a certain Particle goes just by looking at it, so we can check double spends etc